Update the link on the README to the correct v2 website

cdrage opened the pull request:
Updates the README slightly in order to have the correct link to the v2
spec since the previous link 404'd.

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

kadel commented on README.md:
```diff
@@ -2,7 +2,9 @@
-This repository contains the specs, docs and examples of the devfile v1. Devfile v2 specification doc is [here](https://devfile.github.io/website/devfile/index.html) instead. 
+Devfile v1 specs docs and examples are located here.
+
+For Devfile v2, spec, docs and examples are located [here](https://devfile.github.io/devfile/index.html) instead. 
```
Maybe is just me, but that sense looks really weird

cdrage replied:
Woops, I forgot to remove the previous sentence.

sparkoo replied:
it's not only you

cdrage committed:
Update the link on the README to the correct v2 website

Updates the README slightly in order to have the correct link to the v2
spec since the previous link 404'd.

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

sparkoo committed:
Update README.md
